src/axolotl/utils/config/__init__.py

```diff
@@ -240,13 +244,20 @@ def validate_config(cfg: DictDefault, capabilities: Optional[dict] = None):
         ) = merge_input_args()
 
     if capabilities:
+        if env_capabilities is None:
+            env_capabilities = {
+                "torch_version": str(torch.__version__).split("+", maxsplit=1)[0]
+            }
```

Comment:
Is this necessary as you did set it earlier in `load_cfg`? Perhaps in tests I guess?

If so, should we remove the one in `load_cfg`?

Reply:
One caveat is that, in tests, we don't use `load_cfg`, we usually use `validate_cfg`, so this env may not be set. Just a thought for now